src/components/App.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
-import Header from 'components/Header';
+import Landing from 'components/Landing/Landing';
```

Comment:
Instead of having to do `Landing/Landing` what do you thing abt creating an index file in the landing folder and export all the comps from that?

It'll cause bloat but keep our imports cleaner... Im fine with either way for now

Reply:
There might be a lot of components in the future and defining an alias for each one of them might be tedious.